test/integration/webhook/BUILD

```diff
@@ -0,0 +1,38 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Since there's multiple webhooks in the system, maybe this directory should be admission-webhook or dynamic-admission?